### ISSUE TITLE
Attack on titan: wings of freedom and whats error 0x80410136

### ISSUE THREAD

**Issue by NevermindNai1** [blastrock/pkgj#187]
Version of firmware: 3.68
Version of PKGj: v0.32

2 Steps to reproduce the issue

- Getting to Chapter 3 Episode 
- Subjugate the female titan as normal eren
- Beat the female titan as Titan eren

Obtained result: A black screen that doesnt change using the levels BGM
Expected result: A cutscene where Annie was captured. 

1 step to reproduce the issue
-download Attack on titan wings of freedom
-will for some reason display error message 0x80410136

Basically i was playing the game normally and it softlocked. Ive replayed the level 4 times in four different ways. Restarted the system and played it one time out of ghe four times then waited a few hours to see what happens. Im currently redownloading it and it keeps displayimg error 0x80410136

Sorry if it is confusing this is my first time doing this.

**Comment by NevermindNai1** [comment]
Thank you.
